src/components/pages/ClassInfoPage.jsx

```diff
@@ -0,0 +1,207 @@
+          <img src={IconDegree} alt="Icon degree" className="w-6 h-6" />
+          <p className="font-roboto text-xl">{teacherInfo.education}</p>
+        </div>
+        <p className="font-roboto leading-7 text-xl pt-4 w-2/3">
+          {showFullExperience
+            ? teacherInfo.experience
+            : `${teacherInfo.experience.substring(0, 100)}...`}
+        </p>
+        <button
+          onClick={handleToggleExperience}
+          className="text-black font-semibold underline hover:text-darkGreen"
+        >
+          {showFullExperience ? 'Show Less' : 'Read More'}
+        </button>
+      </div>
+    </div>
+  );
+};
+
+const ClassInfoPage = () => {
+  const { token } = useAuth();
+  const { classId } = useParams();
+  const [classItem, setClassItem] = useState(null);
+  const [teacherInfo, setTeacherInfo] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (token) {
+      const fetchClassAndTeacher = async () => {
+        try {
+          // Fetch the class details
+          const classData = await getClassDetails(classId, token);
+          setClassItem(classData.class);
+
+          // Fetch the teacher details
+          if (classData.class.createdBy) {
+            const teacherData = await getUserData(
+              classData.class.createdBy,
+              token
+            );
+            setTeacherInfo(teacherData.data);
+          }
+        } catch (error) {
+          setError(error.message);
+        } finally {
+          setIsLoading(false);
+        }
+      };
+
+      fetchClassAndTeacher();
+    } else {
+      console.error('No token found, user might not be logged in.');
+    }
+  }, [classId, token]);
+
+  if (isLoading) return <Loader />;
+  if (error) return <p>Error: {error}</p>;
+  if (!classItem) return <p>Class not found.</p>;
+
+  return (
+    <div className="container m-auto p-10">
+      <div className="flex flex-col w-full text-black">
+        <div className="flex justify-between flex-col lg:flex-row">
+          <div className=" flex  flex-col  w-full lg:w-[75%] md:flex-row lg:flex-row pb-4">
+            <div className="w-full m:w-[60%] font-roboto text-xl">
+              <h2 className="font-spartan font-medium text-4xl pb-8">
+                {classItem.classTitle}
+              </h2>
+              <p className="w-full md:w-[95%]">{classItem.description}</p>
+            </div>
+
+            <div className="flex w-full lg:w-3/4 pl-0  md:pl-2">
+              <img
+                className="rounded-lg"
+                src={classItem.classImageUrl}
+                alt={classItem.classTitle}
+              />
+            </div>
+          </div>
+
+          <div className="flex flex-row w-full lg:w-[15%] lg:flex-col text-black border-2 border-lightGreen rounded-xl mt-4 mb-8 lg:m-4">
+            <div className="flex flex-row flex-wrap justify-between p-1 lg:flex-col">
```

Comment:
I think there are too many divs. You need one div as a container. Inside the container div you should have 2 divs. The first one with a colored background and rounded corners at the top. The second one with a colored border and rounded corners at the bottom.

Reply:
or you can add rounding to the container div